schema_forms/other_radiology.py

```diff
@@ -0,0 +1,7 @@
+from wtforms import StringField, TextAreaField, SelectField, SubmitField
+from wtforms.fields.html5 import DateField
+from db_dict.common_dict import CommonDict
+from schema_forms.form_utilities import SectionForm
```

Comment:
whats this?

Reply:
I edited the imports to remove the ones which are not being used in this module.